### PR TITLE
Error in console instead of throwing exception

### PR DESCRIPTION
**Pull request by jstnhuang**
Since this is intended to be a warning, we can print a warning to the console instead of throwing an exception. I encountered this exception with some normal-looking point clouds, and the exception caused my entire script execution to halt. I tried this change and the point clouds still rendered properly.

### CODE REVIEW

File: src/sensors/Particles.js [hunk 1]
```diff
@@ -125,6 +125,6 @@ function finishedUpdate(particles, n)
     particles.attribs.alpha.needsUpdate = true;
 
     if(n>particles.max_pts){
-        throw 'Attempted to draw more points than max_pts allows';
+        console.warn('Attempted to draw more points than max_pts allows');
```
Comment (viktorku) [inline src/sensors/Particles.js]: I would make this `console.error` instead actually. Then the effect will be the same, but at least your script execution won't halt

**Comment by jstnhuang** [comment]
Ran grunt build, could you take a look?

**Comment by jihoonl** [comment]
This PR includes unnecessary code-style changes that are irrelevant to `Warn in console instead of throwing exception`. It makes hard to review what this PR brings in. Could you change only relevant part for the PR? I would be happy to review code style change with another PR. :)

**Comment by jstnhuang** [comment]
Sorry, my editor did that. I force pushed an update to this branch without the formatting changes.

**Comment by jihoonl** [comment]
nice. Thanks. ;)

**Pull request merged**